### ISSUE TITLE
missing build dependencies in spec

### ISSUE THREAD

**Issue by anderbubble** [hpcng/warewulf#541]
gpgme-devel

```
cd cmd/config_defaults && go build -ldflags="-X 'github.com/hpcng/warewulf/internal/pkg/warewulfconf.ConfigFile=./etc/warewulf.conf'\
 -X 'github.com/hpcng/warewulf/internal/pkg/node.ConfigFile=./etc/nodes.conf'"\
 -mod vendor -tags "containers_image_openpgp containers_image_ostree" -o ../../config_defaults
go build -o ./wwapid internal/app/api/wwapid/wwapid.go
# github.com/mtrmac/gpgme
vendor/github.com/mtrmac/gpgme/data.go:4:11: fatal error: gpgme.h: No such file or directory
 // #include <gpgme.h>
           ^~~~~~~~~
compilation terminated.
make: *** [Makefile:276: wwapid] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.hD6oM4 (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.hD6oM4 (%build)
```

**Comment by anderbubble** [comment]
Another. libassuan-devel fixed

```
cd cmd/config_defaults && go build -ldflags="-X 'github.com/hpcng/warewulf/internal/pkg/warewulfconf.ConfigFile=./etc/warewulf.conf'\
 -X 'github.com/hpcng/warewulf/internal/pkg/node.ConfigFile=./etc/nodes.conf'"\
 -mod vendor -tags "containers_image_openpgp containers_image_ostree" -o ../../config_defaults
go build -o ./wwapid internal/app/api/wwapid/wwapid.go
# github.com/mtrmac/gpgme
/usr/bin/ld: cannot find -lassuan
collect2: error: ld returned 1 exit status
make: *** [Makefile:276: wwapid] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.NgADLx (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.NgADLx (%build)
```

**Comment by MatthewHink** [comment]
For dnf systems like rocky, add powertools:

`sudo dnf config-manager --set-enabled powertools
sudo dnf install gpgme-devel
sudo dnf install libassuan-devel`

**Comment by anderbubble** [comment]
@MatthewHink I was able to install the packages. My point is that they need to be listed as BuildRequires in the spec file.

**Comment by MatthewHink** [comment]
Makes sense to me. I hit the same issue in January and did not know how to install the dnf packages on Rocky. Docs to do so cannot hurt while we’re here.

**Comment by anderbubble** [comment]
Fixed in #564.

**Comment by bindatype** [comment]
I'm sorry to comment on a closed issue but if someone using Rocky 9 finds this thread they can do the following (powertools installation is slightly different in Rocky 9):

sudo dnf config-manager --enable crb
sudo dnf install gpgme-devel
sudo dnf install libassuan-devel
